logos/fluidcoins/fluidcoins.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="812" height="812" viewBox="0 0 812 812" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Hey man, @adelowo this is a base 64 encoded PNG wrapped as an SVG. Please replace with a high-quality vector version of the logo

Reply:
Resolved in https://github.com/PaystackHQ/nigerialogos/pull/168/commits/c437a60ed1b9c8b211a59a221d8c2f45dcd5f27d . Apologies it took this long :)